lib/utils/logger.cpp

```diff
@@ -0,0 +1,124 @@
+#include "logger.hpp"
+
+#include <chrono>
+
+namespace hyped::utils {
+void Logger::set_level(const Level level)
+{
+  level_ = level;
+};
+
+void Logger::intToLevel(const int level)
+{
+  switch (level) {
+    case 0:
+      level_ = Level::kNone;
+      break;
+    case 1:
+      level_ = Level::kDebug;
+      break;
+    case 2:
+      level_ = Level::kInfo;
```

Comment:
Way too many levels to be practical. At most have fatal, warnings, info, and debug.

Reply:
Yeah that's what i thought too, i just followed ROS's levels but i'll remove kError.

kNone is there for tests when we don't want output to stdout/stderr